src/utils/hooks/useAlerts/utils/utils.ts

```diff
@@ -1,6 +1,3 @@
-import { murmur3 } from 'murmurhash-js';
```

Comment:
do we still use this library somewhere in our code?

Reply:
Yes.

Reply:
We are using it in `generateAlertId` we use in `addAlertIdToRule` I changed to fix the bug. If we try to remove it, I am worried the bug will be back again.

Reply:
generateAlertId i can see is consumed by addAlertIdToRule , so basically this bug happen because of *twice* using murmur3 , so removing it here its good

Reply:
Exactly!

Reply:
Exactly!